cmd/grype/cli/commands/db_update.go

```diff
@@ -26,7 +26,9 @@ func DBUpdate(app clio.Application) *cobra.Command {
 }
 
 func runDBUpdate(opts options.Database) error {
-	dbCurator, err := db.NewCurator(opts.ToCuratorConfig())
+	cfg := opts.ToCuratorConfig()
+	cfg.RequireUpdateCheck = true // always consider an update check failure a db update failure
```

Comment:
Do we want to move this to PreRunE so that the config is printed with this change in debug logging?

Reply:
It turns out the PreRunE didn't work. I think, though, that any `grype db <command>` which happens to run the db update should, by default, consider failure to check for an update to be an update failure, so moved the default configuration to the appropriate place. Note that this does _not_ currently apply to running the normal grype scan, only the `grype db` operations. This also allows users to disable this behavior if they _really_ want to, using an explicitly configured `GRYPE_DB_REQUIRE_UPDATE_CHECK=false`...